src/map/skills/archer/chargearrow.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) rAthena Dev Teams - Licensed under GNU GPL
+// For more information, see LICENCE in the main folder
+
+#pragma once
+
+#include "../weapon_skill_impl.hpp"
+
+#include "../../battle.hpp"
+
+class SkillChargeArrow : public WeaponSkillImpl {
```

Comment:
It is a WeaponSkillImpl, but not removed from the switch in skill_castend_damage_id